internal/forms/forms.go

```diff
@@ -377,7 +377,7 @@ func unzip(src, dest string) error {
 		if f.FileInfo().IsDir() {
 			os.MkdirAll(path, f.Mode())
 		} else {
-			os.MkdirAll(filepath.Dir(path), f.Mode())
+			os.MkdirAll(filepath.Dir(path), 0755)
```

Comment:
You can probably see above that this chunk of code was "heavily inspired" by StackOverflow. The point of f.Mode() is that the zip file has a preferred set of permissions it suggests to use, but I think you're right here, we can simply use 0755.

Can you please do the same 2 lines above for the directory case?

Reply:
lol! I wondered but in testing it didn't seem to affect the "issue" although like I said I have no clue. Change till it works right!

I've updated the branch with that as well. Better?

Reply:
Looks good to me, thanks!